android-app/app/build.gradle

```
@@ -52,7 +52,7 @@ android {
 
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
-    implementation 'com.google.android.material:material:1.8.0'
+    implementation 'com.google.android.material:material:1.7.0'
```

why is this needed?

---

It introduced a but anyways, that the fragment was not visible in the builder. Everything above 1.7.0 does somehow not work with the tablayout which results in render issues

---

Where is the link to get more detail here? Where is the documentation and the tracking issue so we don't loose this information?

---

The problem also exists with 1.9.0 so it was not fixed in the new version. 
The only source you can find online is that some people testet it out manually like here: https://stackoverflow.com/questions/75573385/render-problems-with-tablayout-and-viewholder

---

The best way would be to create a bug in the bugtracker (and check for duplicates before-hand). StackOverflow answers, especially when they are low-quality, shouldn't be used for bug tracking (it is hard to check whether the issue eventually resolved there, as it is not a bug tracker _duh_).

Please put it inline, e.g. 
```suggestion
    implementation 'com.google.android.material:material:1.7.0' // cannot update to anything > 1.7.0 yet because of <insert-bug-link-here>
```

And also create the issue on YouTrack so we don't forget about this. We should avoid relying on old library versions.

---

https://dhbw-karlsruhe.myjetbrains.com/youtrack/issue/FiBo-328/Update-android-studio-and-dependecies
Added this ticket in our backlog